src/core/Form/FilterInput/FilterInput.tsx

```diff
@@ -0,0 +1,186 @@
+import React, { Component, ChangeEvent } from 'react';
+import { default as styled } from 'styled-components';
+import { withSuomifiDefaultProps } from '../../theme/utils';
+import {
+  HtmlInput,
+  HtmlInputProps,
+  HtmlDiv,
+  HtmlDivProps,
+} from '../../../reset';
+import { TokensProp, InternalTokensProp } from '../../theme';
+import { baseStyles } from './FilterInput.baseStyles';
+import classnames from 'classnames';
+import { Omit } from '../../../utils/typescript';
```

Comment:
I think we should stop using the custom Omit for TS, since I guess we anyways require TS 3.8 or later.

Reply:
And I'd prefer having the import ordered to some extent, e.g. React, external libs, utlis styles and resets, other comps, component styles and so on.

Reply:
> And I'd prefer having the import ordered to some extent, e.g. React, external libs, utlis styles and resets, other comps, component styles and so on.

Tried quickly to look if there would be some nice automatic stuff to do this, but didn't find one.

Reply:
These both are now changed and commits pushed.

Reply:
There are tools to handle import order, but I think those use only external - internal and alphabetic order. Could be useful still.